.github/workflows/deploy_tre_reusable.yml

```diff
@@ -436,7 +436,7 @@ jobs:
           COMMAND: |
             # Loop to retry the make command up to 3 times
             for i in {1..3}; do
-            make bundle-build bundle-publish DIR=${{ matrix.BUNDLE_DIR }} && ec=0 && break || ec=\$? && sleep 30;
+              make bundle-build bundle-publish DIR=${{ matrix.BUNDLE_DIR }} && ec=0 && break || ec=\$? && sleep 30;
```

Comment:
Again, is this broken upstream, or just linting?

https://github.com/microsoft/AzureTRE/blob/3fcbdf9c6df25ccd3500702bff952e95766fa6d2/.github/workflows/deploy_tre_reusable.yml#L481

Reply:
I'm afraid it is broken in the upstream 😱

Reply:
The end to end tests still work, so is this just linting?